index.html

```diff
@@ -215,7 +214,50 @@ <h2 class="holder-title">
     <h3 class="last-title-container">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
     </h3>
-    </div>
+    <form action="https://formspree.io/f/{form_id}" method="post">
+         <div class="form-block">
```

Comment:
- [ ] When you will sign up to formspree then you will be assigned a form_id which you will use in the action instead of **{form_id}**. Kindly implement the formspree properly as suggested by the previous reviewer 🤔 

**You can reach out to me on slack by searching my name Hammas in full-time-students channel 👍 for instant connection 😊**